scripts/update.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -e
+
+# Make sure submodule is registered and up-to-date.
+git submodule update --init
+
+# Fetch remote changes.
+cd ckeditor-dev
+git fetch
```

Comment:
If you only `fetch` but don't checkout then the `reset HEAD` below doesn't reset to watch you fetched (you could do that with `git reset --hard FETCH_HEAD` if you wanted).